Add documentation clarification (#206)
dapr/js-sdk: daprdocs/content/en/js-sdk-docs/js-client/_index.md

description: JavaScript Client SDK for developing Dapr applications
---

## Introduction

> The Dapr Client will allow you to perform communication with the Dapr Sidecar and get access to its client facing features such as: Publishing Events, Invoking Output Bindings, State Management, Secret Management, and much more.

Reviewer: ```suggestion
The Dapr Client allows you to communicate with the Dapr Sidecar and get access to its client facing features such as  Publishing Events, Invoking Output Bindings, State Management, Secret Management, and much more.
```

## Installing and importing Dapr's JS SDK

> 1. First install the SDK with `npm`:

Reviewer: ```suggestion
1. Install the SDK with `npm`:
```

```bash

> npm i dapr-client

Reviewer: Should we add `--save`?

```

> 2. Then, Import the libraries:

Reviewer: ```suggestion
2. Import the libraries:
```
